content/nginxaas-google/getting-started/create-deployment/deploy-console.md

```diff
@@ -18,26 +18,15 @@ Before you can deploy NGINXaaS, follow the steps in the [Prerequisites]({{< ref
 
 ### Create a network attachment
 
-NGINXaaS requires a [network attachment](https://cloud.google.com/vpc/docs/about-network-attachments) to connect your NGINXaaS deployment to your VPC network. The network attachment must be created in a region we support.
-
-   {{< call-out "caution" >}}NGINXaaS for Google Cloud currently supports the following regions:
-
-   {{< table "table" >}}
-   |NGINXaaS Geography | Google Cloud Regions |
-   |-----------|---------|
-   | US  | us-west1, us-east1, us-central1 |
-   | EU    | europe-west2, europe-west1 |
-   {{< /table >}}
-
-   {{< /call-out >}}
+NGINXaaS requires a [network attachment](https://cloud.google.com/vpc/docs/about-network-attachments) to connect your NGINXaaS deployment to your VPC network. The network attachment must be created in a region we support. See the [Supported Regions]({{< ref "/nginxaas-google/overview.md#supported-regions" >}}) documentation for the full list of supported regions.
 
 1. Access the [Google Cloud Console](https://console.cloud.google.com/).
-1. Create a consumer VPC network and subnetwork. See [Google's documentation on creating a VPC and subnet](https://cloud.google.com/vpc/docs/create-modify-vpc-networks#console_1) for a step-by-step guide.
+2. Create a consumer VPC network and subnetwork. See [Google's documentation on creating a VPC and subnet](https://cloud.google.com/vpc/docs/create-modify-vpc-networks#console_1) for a step-by-step guide.
```

Comment:
I think this is intentional so that we don't need to constantly update the numbers as we add or remove steps. Using 1. everywhere resolves correctly in markdown when rendered..

Reply:
thanks for catching this 👍